games-roguelike/dwarf-fortress/dwarf-fortress-0.47.04.ebuild

```diff
@@ -1,12 +1,12 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
In EAPI 7 `BDEPEND` should be split from `DEPEND`

Reply:
I add this: `BDEPEND="virtual/pkgconfig"` and remove `"virtual/pkgconfig"` from `DEPEND`.